app.js

```diff
@@ -1,74 +1,88 @@
-require('./config/mongoose')
-const express = require('express')
-const logger = require('morgan')
-const cookieParser = require('cookie-parser')
-const createError = require('http-errors')
-const path = require('path')
-const socket = require('socket.io')
+require("./config/mongoose");
+const express = require("express");
+const logger = require("morgan");
+const cookieParser = require("cookie-parser");
+const createError = require("http-errors");
+const path = require("path");
+const socket = require("socket.io");
+const multer = require("multer");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const fileConstants = require("./config/fileHandlingConstants");
 
-const indexRouter = require('./app/routes/index')
-const authRouter = require('./app/routes/auth')
-const usersRouter = require('./app/routes/user')
-const postRouter = require('./app/routes/post')
-const eventRouter = require('./app/routes/event')
-const shortUrlRouter = require('./app/routes/urlShortner')
-const organizationRouter = require('./app/routes/organisation')
-const commentRouter = require('./app/routes/comment')
-const projectRouter = require('./app/routes/project')
-const notificationRouter = require('./app/routes/notification')
+const indexRouter = require("./app/routes/index");
+const authRouter = require("./app/routes/auth");
+const usersRouter = require("./app/routes/user");
+const postRouter = require("./app/routes/post");
+const eventRouter = require("./app/routes/event");
+const shortUrlRouter = require("./app/routes/urlShortner");
+const organizationRouter = require("./app/routes/organisation");
+const commentRouter = require("./app/routes/comment");
+const projectRouter = require("./app/routes/project");
+const notificationRouter = require("./app/routes/notification");
+const proposalRouter = require("./app/routes/proposal");
 
-const app = express()
-const server = require('http').Server(app)
+const app = express();
+const server = require("http").Server(app);
```

Comment:
Also, your code is violating, esLint rules, as we are using single quotes and no semicolon throughout the whole project. If CI would run then the test cases will fail.